src/Entity/File.php

```diff
@@ -120,6 +120,11 @@ class File extends Entity
      */
     protected $status = self::FILE_NEW;
 
+    /**
+     * ORM\OneToOne(targetEntity=DatasetSubmission::class, mappedBy="File", cascade={"persist", "remove"})
+     */
+    private $datasetSubmission;
```

Comment:
File should NOT have relationship with DatasetSubmission, it SHOULD have a relationship with FileSet

Reply:
This was an overlooked mistake. I'd already updated Fileset but left this in from earlier.